solutions/swb-reference/src/backendAPI.ts

```diff
@@ -105,7 +132,7 @@ const apiRouteConfig: ApiRouteConfig = {
     new AuditService(new BaseAuditPlugin(new AuditLogger(logger))),
     logger,
     new DdbDataSetMetadataPlugin(aws, dataSetPrefix, endPointPrefix),
-    {} as DataSetsAuthorizationPlugin //TODO: REPLACE WITH ACTUAL IMPLEMENTATION ONCE ITS AVAILABLE
+    fakeDataSetsAuthorizationPlugin //TODO: REPLACE WITH ACTUAL IMPLEMENTATION ONCE ITS AVAILABLE
```

Comment:
I actually wouldn't worry about adding this (unless you need it right now). The ticket I'm working on right now fixes it.

Reply:
I'll take it out if tests pass without it!